documentation/releaseNotes/releaseNotes.md

```diff
@@ -1,6 +1,6 @@
 Today we are releasing the next official preview of the `dotnet monitor` tool. This release includes:
 
-- ⚠️ [Here is a breaking change we did and its work item] (#737)
-- [Here is a new feature we added and its work item] (#737)
-
-\*⚠️ **_indicates a breaking change_**
\ No newline at end of file
+- Delete endpoint on startup (#2178)
+- Fix duplicated values for `/metrics` (#2149)
+- Add `CollectLiveMetrics` collection rule action (#2119)
+- Convert logger string format and arguments to native platform string width (#2184)
```

Comment:
This should be removed as it does not impact the release.

Reply:
That's what I figured, but decided it was better to put it here and then remove it than for it to be forgotten. I'll pull it out